examples/csharp/HyperspaceApp/Program.cs

```diff
@@ -0,0 +1,123 @@
+﻿/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Microsoft.Spark.Extensions.Hyperspace;
+using Microsoft.Spark.Extensions.Hyperspace.Index;
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.Sql.Types;
+using System.Collections.Generic;
+
+namespace HyperspaceApp
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Create Spark session.
+            SparkSession spark = SparkSession
+                .Builder()
+                .AppName("Hyperspace example")
+                .Config("spark.some.config.option", "some-value")
```

Comment:
remove this?

Reply:
This is copied from the Scala example:  https://github.com/microsoft/hyperspace/blob/master/examples/scala/src/main/scala/App.scala#L29

Reply:
@apoorvedave1 This follows what Spark examples do. Check [here](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala#L39) as an example.

Reply:
oh interesting. Thanks!